scripts/_common.sh

```diff
@@ -61,9 +64,11 @@ function vpnclient_deploy_files_and_services()
   #=================================================
   # PHP-FPM CONFIGURATION
   #=================================================
-  ynh_print_info "Configuring php-fpm..."
+  ynh_print_info "Configuring PHP-FPM..."
 
-  ynh_add_fpm_config
+  # Create a dedicated PHP-FPM config
+  ynh_add_fpm_config --phpversion=$YNH_PHP_VERSION
+  phpversion=$(ynh_app_setting_get --app=$app --key=phpversion)
```

Comment:
Eh not sure if that's any useful to do this ? Or maybe that's what's used in the nginx conf later ?

Reply:
NGINX is not happy without this...